### PR TITLE
ISSUE-539: Pass file path on DockerSaveImage instead of file

### PR DESCRIPTION
**Pull request by kalap92**

### CODE REVIEW

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerLoadImage.groovy [hunk 1]
```diff
@@ -6,10 +6,11 @@ import org.gradle.api.tasks.Input
 class DockerLoadImage extends AbstractDockerRemoteApiTask {
 
     @Input
-    InputStream imageStream
+    String inputFilePath
```
Comment (cdancy) [inline src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerLoadImage.groovy]: Lets keep the focus of this PR purely on `DockerSaveImage` and remove this change.

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImage.groovy [hunk 2]
```diff
@@ -30,8 +30,10 @@ class DockerSaveImage extends AbstractDockerRemoteApiTask {
 
     @Override
     void runRemoteCommand(Object dockerClient) {
-        if (!destFile || !destFile.isFile()) {
-            throw new GradleException("Invalid destination file: " + destFile)
+        boolean success = destFile.createNewFile()
```
Comment (cdancy) [inline src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImage.groovy]: Think we might want to do something like:
```
if (!destFile.exists()) {
    if(!destFile.createNewFile()) {
        throw new GradleException("Could not create file @ ${destFile.path}")
    }
}

if (destFile.isDirectory()) {
    throw new GradleException("destFile cannot be a directory")
}
```

Comment (kalap92) [reply src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImage.groovy]: Sounds good !

File: src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerLoadImage.groovy (hunk 1 above)
Comment (cdancy) [reply src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerLoadImage.groovy]: @kalap92 did we want to remove this to get the PR merged?

**Comment by cdancy** [comment]
Closing as changes have been merged as part of #552

**Pull request closed without merging**